packages/sdk/src/evm/common/feature-detection.ts

```diff
@@ -294,14 +295,15 @@ async function extractIPFSHashFromBytecode(
   );
 
   // load these lazily to avoid loading them when they are not needed
-  const [cbor, multiHashes] = await Promise.all([
-    import("cbor-web"),
-    import("multihashes"),
-  ]);
+  const [cbor] = await Promise.all([import("cbor-x")]);
```

Comment:
Prob don't need a promise all anymore here. cbor-x allows to import just the decoding too:

`import { decode } from 'cbor-x/decode'`

Reply:
removed it and moved cbor to top